src/main/java/org/jabref/model/entry/BibEntryTypesManager.java

```diff
@@ -170,6 +178,10 @@ private void addCustomOrModifiedType(BibEntryType type) {
             customOrModifiedType.remove(type);
             customOrModifiedType.add(type);
         }
+        
+        private void removeCustomEntryType(BibEntryType type) {
```

Comment:
Please rename this to `CustomOrModified` (I remember that I had troubles understanding the code until I realized that sometimes a distinction between new custom types and modified ones was made, and sometimes not.)